code/game/objects/items/scanners.dm

```diff
@@ -299,75 +311,87 @@ REAGENT SCANNER
 			if(!issynth(patient)) // synthetics don't expire
 				advice += list(list(
 					"advice" = "Time remaining to revive: [DisplayTimeText((TIME_BEFORE_DNR-(patient.dead_ticks))*20)].",
+					"tooltip" = "This is how long until the patient is permanently unrevivable. Stasis bags pause this timer.",
 					"icon" = "clock",
 					"color" = dead_color
 					))
 			if(patient.wear_suit && patient.wear_suit.atom_flags & CONDUCT)
 				advice += list(list(
 					"advice" = "Remove patient's suit or armor.",
+					"tooltip" = "To defibrillate the patient, you need to remove anything conductive obscuring their chest.",
 					"icon" = "shield-alt",
 					"color" = "blue"
 					))
-			if(patient.health + patient.getOxyLoss() + (DEFIBRILLATOR_HEALING_TIMES_SKILL(user.skills.getRating(SKILL_MEDICAL))) >= patient.get_death_threshold() || isrobot(patient))
+			if(ORGANIC_REVIVE || isrobot(patient) || issynth(patient) && SYNTHETIC_REVIVE)
 				advice += list(list(
 					"advice" = "Administer shock via defibrillator!",
+					"tooltip" = "The patient is ready to be revived, defibrillate them as soon as possible!",
 					"icon" = "bolt",
 					"color" = "yellow"
 					))
 		if(issynth(patient) || isrobot(patient)) // robotic damage advice
 			if(patient.getBruteLoss() > 0)
 				advice += list(list(
 					"advice" = "Use a blowtorch or nanopaste to repair the dented areas.",
+					"tooltip" = "Only a blowtorch or nanopaste can heal scorched robotic limbs. Other sources, such as chemicals, will not work.",
 					"icon" = "tools",
 					"color" = "red"
 					))
 			if(patient.getFireLoss() > 0)
 				advice += list(list(
 					"advice" = "Use a cable coil or nanopaste to repair the scorched areas.",
+					"tooltip" = "Only cable coils or nanopaste can heal scorched robotic limbs. Other sources, such as chemicals, will not work.",
 					"icon" = "plug",
 					"color" = "orange"
 					))
 		else // organic damage advice
 			if(patient.getBruteLoss() > 20)
 				advice += list(list(
 					"advice" = "Use trauma kits or sutures to repair the lacerated areas.",
+					"tooltip" = "Advanced trauma kits will heal brute damage, scaling with how proficient you are in the Medical field. Patients may also self-administer gauze on bleeding wounds, allowing them to slowly heal on their own.",
 					"icon" = "band-aid",
 					"color" = "green"
 					))
 			if(patient.getFireLoss() > 20)
 				advice += list(list(
 					"advice" = "Use burn kits or sutures to repair the burned areas.",
+					"tooltip" = "Advanced burn kits will heal burn damage, scaling with how proficient you are in the Medical field. Patients may also self-administer ointment on bleeding wounds, allowing them to slowly heal on their own.",
 					"icon" = "band-aid",
 					"color" = "orange"
 					))
```

Comment:
As mentioned above, if you just have a 'issynth/meatboi' var, you can just have these organised as ternaries for advice/tooltip/icon, rather than have these big copy pasted if chains